index.js

```diff
@@ -1,7 +1,7 @@
 Cypress.Commands.add('injectAxe', () => {
   cy.window({ log: false }).then(window => {
     const axe = require('axe-core')
-    window.eval(axe.source)
```

Comment:
Is there a reason the original code ran `window.eval(axe.source)` instead of `window.axe = axe`?